menu/server_configuration.sh

```diff
@@ -48,7 +48,7 @@ choice=$(whiptail --title "$TITLE" --checklist \
 $CHECKLIST_GUIDE\n\n$RUN_LATER_GUIDE" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 "Static IP" "(Set static IP in Ubuntu with netplan.io)" OFF \
 "Security" "(Add extra security based on this http://goo.gl/gEJHi7)" OFF \
-"deSEC" "(Automatically get a domain from desec.io, configure ddclient and activate TLS)" "$STARTUP_SWITCH" \
+"deSEC" "(Automatically setup a dedyn.io domain, together with DDNS and TLS)" "$STARTUP_SWITCH" \
```

Comment:
Here the same for the word `setup` -> should be `set up`, imo

Reply:
Great feeback, please change to set up 

https://writingexplained.org/setup-vs-set-up-difference